.github/workflows/canister-tests.yml

```
@@ -250,29 +260,39 @@ jobs:
         uses: actions/download-artifact@v2
         with:
           name: test_app.wasm
-          path: .
+          path: demos/test-app
 
-      - name: Prepare Canister WASM Files
+      - name: Deploy Internet Identity
         run: |
-          mv internet_identity_test.wasm docker-test-env/dfx-docker/canisters/internet_identity.wasm
-          cp src/internet_identity/internet_identity.did docker-test-env/dfx-docker/canisters/internet_identity.did
-          mv test_app.wasm docker-test-env/dfx-docker/canisters/test_app.wasm
-          cp demos/test-app/test_app.did docker-test-env/dfx-docker/canisters/test_app.did
+          dfx canister create --all
+          dfx canister install internet_identity --argument '(null)' --wasm internet_identity_test.wasm
```

```suggestion
          dfx canister install internet_identity --wasm internet_identity_test.wasm
```

this might or might not work; for `dfx deploy` at least and since 0.10 you can omit null args